src/ddspdrum/defaults.py

```diff
@@ -0,0 +1,7 @@
+"""
+Default parameters
+"""
+
```

Comment:
These are *defaults*, not fixed global parameters. I can imagine circumstances in which someone might want to have several synths running at once with different sample rates (e.g. to work with samples of multiple resolutions), so I have rewritten to allow this.

Reply:
Nice, I think it is good to have the rates passed in during construction.